Gemfile

```diff
@@ -1,8 +1,14 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
+# Para el laburo
 ruby '2.5.1'
 
+# Para casa
+# ruby '2.5.0'
```

Comment:
The version of ruby should not change in your application depending on the computer you are using. The reverse behaviour is expected. Adapt the ruby version on your computador to the project. For that reason we use `rbenv`